src/lib/CopyBSR.svelte

```diff
@@ -0,0 +1,101 @@
+<script lang="ts">
+  import Fa from 'svelte-fa'
+  import { faTwitch } from '@fortawesome/free-brands-svg-icons'
+  import { onMount } from 'svelte'
+  import { fade } from 'svelte/transition'
+
+  export let fontSize: string = '1rem'
+  export let mapId: string
+
+  // Adding props to allow these to be customizable for the MOTW Section
+  export let tooltipTop: string = '32px'
+  export let tooltipLeft: string = '92%'
+  export let tooltipTransform: string = 'translateX(-50%)'
+
+  // Margins because it looks off on the MapCards without it
+  export let marginRight: string = '4.3px'
+  export let marginTop: string = '3px'
+
+  let showTooltip = false
+  let tooltipTimeout: NodeJS.Timeout
+
+  function copyBSR() {
+    const textToCopy = `!bsr ${mapId}`
+    navigator.clipboard
+      .writeText(textToCopy)
+      .then(() => {
+        showTooltip = true
+        clearTimeout(tooltipTimeout)
+        tooltipTimeout = setTimeout(() => {
+          showTooltip = false
+        }, 3000)
```

Comment:
Smart! And works.

What you could consider is a logic that stops showing the tooltip, when your mouse leaves the element. You can use `on:mouseout` event for that on your button.

This solution works to, though. Do consider the map cards though - in that case if you click on the button, leave the map cards with your mouse and then reenter it, it will still be there. But it's nothing severe.

Reply:
Great idea! I was unsure about how to go about this as I wanted to give them enough time to read it, but if they take their mouse off it they've likely read it or can dismiss it much quicker if they want to. Gives the end user more flexibility. Will implement.